glide.lock

```diff
@@ -194,24 +190,32 @@ imports:
 - name: github.com/elazarl/go-bindata-assetfs
   version: 30f82fa23fd844bd5bb1e5f216db87fd77b5eb43
 - name: github.com/emicklei/go-restful
-  version: 892402ba11a2e2fd5e1295dd633481f27365f14d
+  version: 89ef8af493ab468a45a42bb0d89a06fccdd2fb22
   subpackages:
   - log
   - swagger
 - name: github.com/fatih/color
   version: 9131ab34cf20d2f6d83fdc67168a5430d1c7dc23
 - name: github.com/gambol99/go-marathon
-  version: 15ea23e360abb8b25071e677aed344f31838e403
```

Comment:
I've been trying to add:
```
github.com/go-kit/kit/metrics/dogstatsd
github.com/go-kit/kit/metrics/statsd
github.com/go-kit/kit/metrics/multi
github.com/go-kit/kit/log
github.com/go-kit/kit/util/conn
github.com/stvp/go-udp-testing
```
Would greatly appreciate if you could see if it will work on your end.